tools/pyboard.py

```diff
@@ -580,7 +585,7 @@ def execbuffer(buf):
         pyb.exit_raw_repl()
 
     # if asked explicitly, or no files given, then follow the output
-    if args.follow or (args.command is None and not args.filesystem and len(args.files) == 0):
+    if not args.no_follow and (args.follow or (args.command is None and not args.filesystem and len(args.files) == 0)):
```

Comment:
I don't think this line needs to have `no_follow` added.  This logic here will only execute if `--follow` is explicitly passed, or if nothing is passed, ie `$ pyboard.py` on its own.  So it'd only be useful for `$ pyboard.py --no-follow` which is a no-op.